libraries/tuttle/src/tuttle/host/graph/GraphExporter.tcc

```diff
@@ -5,23 +5,27 @@
 #include <boost/graph/graphviz.hpp>
 #include <iostream>
 
-namespace tuttle {
-namespace host {
-namespace graph {
+namespace tuttle
+{
```

Comment:
There is an example here: https://github.com/simulationcraft/simc/blob/master/.clang-format#L74. But the issue is that this feature is available since clang-format-3.7.0, which is slightly new... Currently we are using basic features available since a long time...